justfile

```diff
@@ -38,22 +46,30 @@ anvil *args:
     docker run -p 127.0.0.1:8545:8545 ghcr.io/foundry-rs/foundry:latest "anvil {{args}}"
 
 test *args:
-	@echo 'Omitting slow tests. Use `test-slow` for those. Or `test-all` for all tests.'
-	cargo nextest run --locked --workspace --all-features --verbose {{args}}
+    @echo 'Omitting slow tests. Use `test-slow` for those. Or `test-all` for all tests.'
+    @echo 'features: "all"'
+    cargo nextest run --locked --workspace --all-features --verbose {{args}}
+    cargo nextest run --locked --workspace --verbose {{args}}
```

Comment:
Why are we running all the tests twice now?

Reply:
one uses sqlite for tests using database and the other uses postgres